portfolio/src/main/webapp/index.html

```diff
@@ -188,6 +188,12 @@
           </a>
         </div>
       </div>
+
+      <div id="additional-functional">
```

Comment:
`functional` is an adjective, not a noun.
You can call it `additional-functionality`, but it's better to have a more descriptive name, that explains what exactly is presented in this `div`.